### PR TITLE
💄 UI updates for Primitive NFT 

### PR DESCRIPTION
**Pull request by nwingt**
image
image

### CODE REVIEW

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/_id/index.vue [hunk 1]
```diff
@@ -286,13 +166,13 @@ export default {
         if (this.$route.hash === this.creatorFollowSectionHash) {
           this.$nextTick(this.scrollToCreatorFollowSection);
         } else if (
-          this.currentTab !== portfolioMixin.tabOptions.created &&
-          !this.sortedCollectedNFTs.length
+          !this.isCurrentTabCreated &&
+          // NOTE: Seems computed property `this.nftClassListOfCollected` is not reflecting the actual state
+          !this.nftClassListMap?.collected.length
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/pages/_id/index.vue:171]: Tried, it doesn't work neither

File: src/util/nft.js [hunk 2]
```diff
@@ -152,8 +148,30 @@ export function isValidHttpUrl(string) {
...
+  BookNFT: 'book-nft',
+};
+
+export function getNFTClassCollectionType(classMetadata) {
+  switch (classMetadata?.nft_meta_collection_id) {
+    case 'likerland_writing_nft':
+      return nftClassCollectionType.WritingNFT;
+
+    case 'nft_book':
+    case 'ckxpress_book_nft':
```
Comment (nwingt) [inline src/util/nft.js:162]:
```suggestion
```

**Review by williamchong — APPROVED** [review]

File: src/pages/_id/index.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamchong) [reply src/pages/_id/index.vue:171]: strange, should spend time to investigate when free

File: src/mixins/nft.js [hunk 3]
```diff
@@ -100,8 +107,11 @@ export default {
     NFTClassMetadata() {
       return this.getNFTClassMetadataById(this.classId) || {};
     },
-    isWritingNFT() {
-      return isWritingNFT(this.NFTClassMetadata);
+    nftClassCollectionType() {
+      return getNFTClassCollectionType(this.NFTClassMetadata);
+    },
```
Comment (WeiJun0827) [inline src/mixins/nft.js:112]: Seems some conditional checks in `if (this.isWritingNFT)` should be refactored to `if (this.nftClassCollectionType === nftClassCollectionType.WritingNFT)`?
like [here](https://github.com/likecoin/liker-land/blob/decbf2c249742fa0f073b42c28f77d368a6ddb5c/src/mixins/nft.js#L392)

**Pull request merged**